megazords/ios/MozillaAppServicesTests/FxAccountManagerTests.swift

```diff
@@ -203,12 +203,12 @@ class FxAccountManagerTests: XCTestCase {
         class MockAccount: MockFxAccount {
             override func ensureCapabilities(supportedCapabilities _: [DeviceCapability]) throws {
                 notifyAuthError()
-                throw FirefoxAccountError.unauthorized(message: "Your token is expired yo.")
+                throw FxaError.Authentication(message: "Your token is expired yo.")
             }
 
-            override func checkAuthorizationStatus() throws -> IntrospectInfo {
+            override func checkAuthorizationStatus() throws -> AuthorizationInfo {
                 _ = try super.checkAuthorizationStatus()
-                return IntrospectInfo(active: false)
```

Comment:
`IntrospectInfo` was really not a good name for the public API, where there is no operation called "introspect".